admin/src/templates/forms_xml.html

```diff
@@ -25,7 +25,12 @@
 
 <div class="section">
   <fieldset>
-    <legend translate>Installed Forms</legend>
+    <legend translate>installed.app.forms</legend>
+    <div class="row selection-heading">
+      <div class="col-sm-2" translate>form.icon</div>
+      <div class="col-sm-5" translate>form.id</div>
```

Comment:
These column widths must match the column widths below otherwise the headers don't line up with the content. The first one should be `col-sm-1` and the second `col-sm-6`. If `col-sm-1` is too small for the text without line wrapping then change the content to be 2, 5, and 5.